pkg/tsdb/cloudwatch/models/request/dimension_keys_request.go

```diff
@@ -0,0 +1,57 @@
+package request
+
+import (
+	"net/url"
+
+	"github.com/grafana/grafana/pkg/tsdb/cloudwatch/constants"
+)
+
+type DimensionKeysRequestType uint32
+
+const (
+	StandardDimensionKeysRequest DimensionKeysRequestType = iota
+	FilterDimensionKeysRequest
+	CustomMetricDimensionKeysRequest
+)
+
+type DimensionKeysRequest struct {
+	*ResourceRequest
```

Comment:
DimensionKeyRequest composed with ResourceRequest. All resource requests (think there are 10 in total, but only two migrated to new pattern so far) will be composed with ResourceRequest, as all of them need to define a region.